### ISSUE TITLE
Have entered the xyz of a molecule, but it shows InputError.

### ISSUE THREAD

**Issue by NtuCheRoy** [ReactionMechanismGenerator/ARC#200]
I try to run a freq job for hydrogen giving xyz as input. However, it shows that xyz is not be defined. It is so weird... Below are my input file and trace message. Could you help me solve this problem?  Thank you so much.
By the way, when I run single point job, it doesn't appear this error.

```
project: arc_demo_1

ess_settings:
        qchem:
        - local

job_types:
        conformers: false
        opt: false
        fine: false
        freq: true
        sp: false
        1d_rotors: false
        orbitals: false

level_of_theory: b3lyp/6-31g*
conformer_level: b3lyp/6-31g*
scan_level: b3lyp/6-31g*
ts_guess_level: b3lyp/6-31g*
calc_freq_factor: false

species:
- label: hydrogen
  xyz:
  - |
    H         -4.12347       -0.08887        0.00000
    H         -3.70381        0.48134        0.00000
```
```
Using Theano backend.
ARC execution initiated on Wed Sep 18 11:51:40 2019

###############################################################
#                                                             #
#                 Automatic Rate Calculator                   #
#                            ARC                              #
#                                                             #
#   Version: 1.1.0                                            #
#                                                             #
###############################################################

The current git HEAD for ARC is:
    9fc5c7722322bc0a0b74c82652cc8401f8fa3fae
    Mon Sep 16 22:39:52 2019 -0400


Starting project arc_demo_1

Considering the following job types: ['freq']


Using the following ESS settings:
{'qchem': ['local']}



Warning: Not using a fine grid for geometry optimization jobs




Warning: Not running rotor scans. This might compromise finding the best conformer, as dihedral angles won't be corrected. Also, entropy won't be accurate.


Using b3lyp/6-31g* for refined conformer searches (after filtering via force fields)
Using b3lyp/6-31g* for TS guesses comparison of different methods
Using b3lyp/6-31g* for geometry optimizations
Using b3lyp/6-31g* for frequency calculations
Using b3lyp/6-31g* for single point calculations
Using b3lyp/6-31g* for rotor scans



Warning: Could not determine appropriate Model Chemistry to be used in Arkane for thermochemical parameter calculations.
Not using atom energy corrections and bond additivity corrections!


Using b3lyp/6-31g*//b3lyp/6-31g* as a model chemistry in Arkane
Warning: :root:No frequency scaling factor found for model chemistry b3lyp/6-31g*. Assuming a value of unity. This will affect the partition function and all quantities derived from it (thermo quantities and rate coefficients).
Could not determine the harmonic frequencies scaling factor for b3lyp/6-31g* from Arkane.
Not calculating it, assuming a frequencies scaling factor of 1.


Considering species: hydrogen
<Molecule "[H][H]">


Only one conformer is available for species hydrogen, using it as initial xyz
The only conformer for species hydrogen was found to be isomorphic with the 2D graph representation [H][H]

Traceback (most recent call last):
  File "/global/homes/s/shcheng/ARC/ARC.py", line 76, in <module>
    main()
  File "/global/homes/s/shcheng/ARC/ARC.py", line 70, in main
    arc_object.execute()
  File "/global/u2/s/shcheng/ARC/arc/main.py", line 718, in execute
    dont_gen_confs=self.dont_gen_confs)
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 391, in __init__
    self.schedule_jobs()
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 407, in schedule_jobs
    self.run_conformer_jobs()
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 788, in run_conformer_jobs
    self.process_conformers(label)
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 1385, in process_conformers
    self.run_freq_job(label)
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 872, in run_freq_job
    level_of_theory=self.freq_level, job_type='freq')
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 683, in run_job
    directed_dihedral=directed_dihedral)
  File "/global/u2/s/shcheng/ARC/arc/job/job.py", line 217, in __init__
    raise InputError('{0} Job of species {1} got None for xyz'.format(self.job_type, self.species_name))
arc.arc_exceptions.InputError: freq Job of species hydrogen got None for xyz
```

**Comment by alongd** [comment]
Very strange indeed.
Do you mind updating ARC? I recently changed the way XYZ is treated and represented. If you still have this issue, I'll try to debug it.

**Comment by NtuCheRoy** [comment]
I have updated it, but still appear the error. I also try to input the species by SMILE and it have the same result. Below is the result I run with xyz after update it.
```
Considering species: hydrogen
<Molecule "[H][H]">


Only one conformer is available for species hydrogen, using it as initial xyz
The only conformer for species hydrogen was found to be isomorphic with the 2D graph representation [H][H]

Traceback (most recent call last):
  File "/global/homes/s/shcheng/ARC/ARC.py", line 76, in <module>
    main()
  File "/global/homes/s/shcheng/ARC/ARC.py", line 70, in main
    arc_object.execute()
  File "/global/u2/s/shcheng/ARC/arc/main.py", line 720, in execute
    dont_gen_confs=self.dont_gen_confs)
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 392, in __init__
    self.schedule_jobs()
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 408, in schedule_jobs
    self.run_conformer_jobs()
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 789, in run_conformer_jobs
    self.process_conformers(label)
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 1387, in process_conformers
    self.run_freq_job(label)
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 873, in run_freq_job
    level_of_theory=self.freq_level, job_type='freq')
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 684, in run_job
    directed_dihedral=directed_dihedral)
  File "/global/u2/s/shcheng/ARC/arc/job/job.py", line 180, in __init__
    self.xyz = check_xyz_dict(xyz)
  File "/global/u2/s/shcheng/ARC/arc/species/converter.py", line 320, in check_xyz_dict
    raise TypeError('Expected a dictionary, got {0}'.format(type(xyz_dict)))
TypeError: Expected a dictionary, got <type 'NoneType'>
```

**Comment by alongd** [comment]
The problem with the first issue is indeed a bug, I'll address it soon.

For the second case, I'd like to understand what your expectations are. If you just give SMILES, what 3D structure (i.e., at what level?) would you like to calculate frequencies on when not optimizing? This is also a question we asked ourselves (what users might want), your input would be helpful. My current thought is to through a clearer error message, but not to run frequencies with out an optimized (or user supplied) 3D structure.

**Comment by alongd** [comment]
closed by #208
